CONTRIBUTING.md

```diff
@@ -14,11 +14,11 @@ Here are our guidlines for contributing:
 
 ## <a name="coc"></a> Code of Conduct
 
-Please help us keep MLAPI open and inclusive. Read and follow our [Code of Conduct](CODE_OF_CONDUCT.md).
+Please help us keep Unity Netcode for GameObjects open and inclusive. Read and follow our [Code of Conduct](CODE_OF_CONDUCT.md).
 
 ## <a name="ways"></a> Ways to Contribute
 
-There are many ways in which you can contribute to the MLAPI.
+There are many ways in which you can contribute to the Netcode for GameObjects.
```

Comment:
```suggestion
There are many ways in which you can contribute to Netcode for GameObjects.
```
Nit: not sure whether I should be the one correcting the grammar here we will have this problem all over our code comments as well.